ttnn/ttnn/operations/binary.py

```diff
@@ -480,6 +480,7 @@ def add_and_apply_activation(
     activation: Optional[str] = None,
     memory_config: ttnn.MemoryConfig = ttnn.DRAM_MEMORY_CONFIG,
     dtype: Optional[ttnn.DataType] = None,
+    out_in_place: Optional[bool] = False,
```

Comment:
Could we change this to `in_place` and add documentation for this function parameter?